pages/agent/v3/help/_lock_acquire.md

```diff
@@ -31,21 +31,26 @@ with the `agent-api` experiment enabled.
 ### Examples
 
 ```shell
-$ token=$(buildkite-agent lock acquire llama)
-$ critical_section()
-$ buildkite-agent lock release llama "${token}"
+#!/bin/bash
+token=$(buildkite-agent lock acquire llama)
+# your critical section here...
```

Comment:
So in bash, you can't call a function using parens, bash thinks you're trying to define a new function if you do this:
```shell
[narthana@quirinus docs]$ f()
> {
> echo hello
> }
[narthana@quirinus docs]$ f
hello
[narthana@quirinus docs]$ f()
>
```

Reply:
Good point